### DISCUSSION TITLE
Is there a way to avoid `require("plugin-name").setup()` for plugins that require it?

### DISCUSSION THREAD

**Discussion by sourabhxyz** [LazyVim/LazyVim#761] [General]
Hi, to use [`symbols-outline.nvim`](https://github.com/simrat39/symbols-outline.nvim), I inserted it in table of plugins:

```
{
  "simrat39/symbols-outline.nvim",
},
```
And then for it to work properly, I also added in `/lua/config/lazy.lua` file the line: `require("symbols-outline").setup()`. They mention about calling setup in their README [here](https://github.com/simrat39/symbols-outline.nvim#setup). If I don't do it, I get the error as highlighted in this issue: https://github.com/simrat39/symbols-outline.nvim/issues/213.

But was it really required to do `require("...").setup()`? Is there a way to avoid it by using `lazy.nvim`?

Thanks!

**Comment by dpetka2001** [comment]
Calling `opts={}` or `config=true` is the same as calling the plugin's `setup` function.

**Reply by sourabhxyz** [reply]
Thanks so much @dpetka2001!
